test/e2e/unmanaged_test.go

```diff
@@ -41,14 +44,22 @@ func TestUnmanaged(t *testing.T) {
 	})
 	testframework.MustEnsureImageRegistryIsAvailable(t, client)
 
-	cr, err := client.ImageRegistries().Get(testframework.ImageRegistryName, metav1.GetOptions{})
-	if err != nil {
-		t.Fatal(err)
-	}
+	var cr *imageregistryapi.ImageRegistry
+	var err error
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		cr, err = client.ImageRegistries().Get(testframework.ImageRegistryName, metav1.GetOptions{})
+		if err != nil {
+			t.Fatal(err)
+		}
 
-	cr.Spec.ManagementState = operatorapi.Unmanaged
+		cr.Spec.ManagementState = operatorapi.Unmanaged
```

Comment:
What can concurrently update our CR after the registry is completely deployed?

Reply:
this may have been due to the hot loop we had that was updating the cr and making api calls, might be able to set this back after this pr merges